homeassistant/components/reolink/__init__.py

```diff
@@ -286,6 +286,11 @@ def migrate_entity_ids(
             )
             continue
 
+        # Can be removed in HA 2025.2.0
+        if entity.domain == "switch" and entity.unique_id.endswith("_hdr"):
+            entity_reg.async_remove(entity.entity_id)
+            continue
```

Comment:
Should we keep these alive for 6 months and raise an issue when added/used? We can remove them if someone disabled them

Reply:
> We can remove them if someone disabled them

I don't think that will work: the swich entity will not be set-up anymore, so it will always be unavailable.
If we do set-up the switch enity, removing it here does not work because it will be re-added in the next step when the platforms are set-up.

We could keep both entities around for 6 months, but it is a bit confusing to have 2 entities doing the same thing for new users.
Besides, because my code uses the same class for all switch entities it is a bit of a pain to implement deprication code (raising the issue) specific for 1 entity.

I guess I could make a seperate list of depricated entities and during setup check if it is enabled or not and only set it up if it is enabled and then raise a repair-issue. That would be the nicest, but also the most work to code..... hahaha

Reply:
You can set up both entities, but only if the switch has been added before as it is not disabled yet.

Reply:
I will write some code

Reply:
@joostlek alright, finished, could you take a look if this looks good to you?
Still need to test when I get back home tonight, so do not merge yet.
But you can already approve if you think this is alright.